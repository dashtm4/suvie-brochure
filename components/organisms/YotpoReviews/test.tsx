import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { YotpoReviews } from '.'

describe('<YotpoReviews />', () => {
  it('should renders correct <YotpoReviews />', () => {
    const wrapper = shallow(<YotpoReviews />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
