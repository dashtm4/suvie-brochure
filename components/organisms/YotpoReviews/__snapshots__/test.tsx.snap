// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<YotpoReviews /> should renders correct <YotpoReviews /> 1`] = `
<div
  className="yotpo yotpo-main-widget"
  data-product-id="1"
>
  <div
    className="text-center text-3xl"
  >
    Loading...
  </div>
</div>
`;
